lib/mintel/dashboards/components/panels/prometheus.libsonnet

```diff
@@ -753,6 +753,8 @@ local seriesOverrides = import 'components/series_overrides.libsonnet';
       span=span,
       decimals=null,
       fill=1,
+      min=null,
+      yAxisLabel='Number of Series',
```

Comment:
Would it be worth using `number of timeseries' ?

I just see this being used elsewhere here quite a lot. I guess if we mix widgets on the dashboard it would look more consistent.

Reply:
done